services/accounting/controllers/send.ts

```diff
@@ -114,7 +116,7 @@ const sendKeysend = asyncHandler<
           });
 
           return {
-            // this is not really true if the payment is in flight
+            // this is not really true if the payment is in flight, but we report it as true to the client
             success: true,
             msatAmount: request.msatAmount,
             pubkey: request.pubkey,
```

Comment:
It might help if we send more useful data to the client so this is clearer. For example, using our standard response format (couldn't suggest full response but this is the main idea):
```suggestion
            success: true,
            message: "Keysend payment is in flight",
            data: { msatAmount: request.msatAmount, pubkey: request.pubkey,
            ...
```